### PR TITLE
doc: known_issues: Add HID configurator issue on Linux

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds description of HID configurator issue under Linux.

Jira: NCSDK-8304

### CODE REVIEW

**Comment by MarekPieta** [comment]
Waiting for review from @pdunaj

File: doc/nrf/known_issues.rst
Comment (greg-fer) [inline doc/nrf/known_issues.rst:323]:
```suggestion
NCSDK-8304: HID configurator issues for peripherals connected over Bluetooth LE to Linux host
```

File: doc/nrf/known_issues.rst
Comment (greg-fer) [inline doc/nrf/known_issues.rst:325]:
```suggestion
  In such case, the device will provide HID input reports, but it cannot be configured with the HID configurator.
```

File: doc/nrf/known_issues.rst
Comment (greg-fer) [inline doc/nrf/known_issues.rst:327]:
```suggestion
  **Workaround:** Connect the nRF Desktop peripheral through USB or using the nRF Desktop dongle.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Couple of edits.

**Comment by MarekPieta** [comment]
I squashed the changes

**Review by umapraseeda — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

**Pull request merged**
